src/data/automation_i18n.ts

```diff
@@ -157,35 +175,38 @@ const tryDescribeTrigger = (
       ? describeDuration(hass.locale, trigger.for)
       : undefined;
 
-    if (trigger.above && trigger.below) {
+    if (trigger.above !== undefined && trigger.below !== undefined) {
       return hass.localize(
         `${triggerTranslationBaseKey}.numeric_state.description.above-below`,
         {
           attribute: attribute,
-          entity: entity,
+          entity: formatListWithAnds(hass.locale, entities),
+          numberOfEntities: entities.length,
           above: trigger.above,
           below: trigger.below,
           duration: duration,
         }
       );
     }
-    if (trigger.above) {
+    if (trigger.above !== undefined) {
       return hass.localize(
         `${triggerTranslationBaseKey}.numeric_state.description.above`,
         {
           attribute: attribute,
-          entity: entity,
+          entity: formatListWithAnds(hass.locale, entities),
+          numberOfEntities: entities.length,
           above: trigger.above,
           duration: duration,
         }
       );
     }
-    if (trigger.below) {
+    if (trigger.below !== undefined) {
       return hass.localize(
         `${triggerTranslationBaseKey}.numeric_state.description.below`,
         {
           attribute: attribute,
-          entity: entity,
+          entity: formatListWithAnds(hass.locale, entities),
```

Comment:
I think this should be or? Because only one the entities have to match the trigger and not all?

Reply:
Documentation is not clear about this I think

Reply:
It's indeed an or condition.